Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -320,31 +316,33 @@
     }
 });
 
-Cesium.knockout.getObservable(viewModel, 'rate').subscribe(
+Cesium.knockout.getObservable(viewModel, 'emissionRate').subscribe(
     function(newValue) {
-        particleSystem.rate = parseFloat(newValue);
+        particleSystem.emissionRate = parseFloat(newValue);
     }
 );
 
 Cesium.knockout.getObservable(viewModel, 'particleSize').subscribe(
     function(newValue) {
         var particleSize = parseFloat(newValue);
-        particleSystem.minimumWidth = particleSize;
-        particleSystem.maximumWidth = particleSize;
-        particleSystem.minimumHeight = particleSize;
-        particleSystem.maximumHeight = particleSize;
+        if (particleSystem.imageSize) {
+            particleSystem.imageSize.x = particleSize;
+            particleSystem.imageSize.y = particleSize;
+        } else {
+            particleSystem.imageSize = new Cesium.Cartesian2(particleSize, particleSize);
+        }
```

Comment:
This is still broken. `imageSize` will never be defined- it's not a property of the class. (Additionally you should avoid creating a new Cartesian2 each update).

Replacing the function body with this should work:
```
       particleSystem.minimumImageSize.x = newValue;
       particleSystem.minimumImageSize.y = newValue;
       particleSystem.maximumImageSize.x = newValue;
       particleSystem.maximumImageSize.y = newValue;
```